wherehows-web/app/templates/components/dataset-author.hbs

```diff
@@ -1,6 +1,15 @@
 <td>
   {{user-avatar userName=owner.userName}}
+
   {{owner.userName}}
+
+  {{#if isOwnerInActive}}
+
+    <button class="nacho-button nacho-button--small dataset-author-record__action--inactive">
+      Inactive
```

Comment:
On another note, there's no action tied to this button. Assuming this component is incomplete?